docs/DeveloperGuide.md

```diff
@@ -484,16 +485,16 @@ Those without any stars are user stories that were considered but will not be im
 |  `* *`   | first-time user           | see a help message showing all the commands/feature I can use                                                        | I can try out all the different features by referring to the message.                                                                                               |
-|  `* *`   | lazy user                 | have the ability to bypass certain controlled features                                                               |                                                                                                                                                                     |
-|   `*`    | user                      | assign tasks to contacts                                                                                             | I can track which tasks have been assigned to each contact.                                                                                                         |
-|   `*`    | user                      | update the status of tasks of contacts                                                                               | I can track the status of completion of the tasks assigned to contacts.                                                                                             |
+|  `* *`   | lazy user                 | want to use command keyword shortcuts                                                                                | I can quickly execute commands without needing to type the full command word.                                                                                       |
+|  `* *`   | user                      | assign tasks to certain contacts                                                                                     | I can track which tasks have been assigned to each contact.                                                                                                         |
+|  `* *`   | user                      | update the status of tasks of contacts                                                                               | I can track the status of completion of the tasks assigned to contacts.                                                                                             |
+|  `* *`   | busy user                 | add multiple wedding events                                                                                          | I can track contacts for multiple weddings at once.                                                                                                                 |
+|  `* *`   | busy user                 | tag each contact to a wedding                                                                                        | I can easily see which contacts are relevant to each wedding.                                                                                                       |
```

Comment:
Maybe here for the "I can", can say "assign each..." just to avoid confusion with tags?